service/src/main/java/bio/terra/workspace/service/workspace/flight/cloud/gcp/DeleteGcpProjectStep.java

```diff
@@ -60,18 +61,17 @@ public StepResult undoStep(FlightContext flightContext) throws InterruptedExcept
       return StepResult.getStepResultSuccess();
     }
     // Do not attempt to undo project deletions.
-    // TODO: investigate recovering projects if this happens often and recovery works as a "undo."
-    String projectId = cloudContext.get().getGcpProjectId();
-    logger.error("Unable to undo deletion of project [{}]", projectId);
+    logger.error("Unable to undo deletion of project");
     return flightContext.getResult();
```

Comment:
With the stairway fix merged, this can just return a FATAL step result with the error instead of `flightContext.getResult()`